regular-expressions/introduction/basic-syntax-i/regex-wildcards-ii.md

```diff
@@ -18,56 +18,62 @@ aspects:
 ---
 ## Content
 
-Going back to our previous example - if we wanted to find all names beginning with `Fre` then we could write our regex pattern as:
-
-`/Fre\w*/`
-
-This would match the names that start with `Fre`, and then have zero or more word characters. 
+Going back to our previous example - if we wanted to find all names beginning with 'Fre' then we could write our regex pattern as:
 
 ```
-'Fred' ✅
-'Freya' ✅
-'Francis' ❌
+/Fre\w*/
 ```
 
-Here are a few more wildcards that can be used when you want to be more specific than using a `.`, `\w` or `\W`!
+This would match the names that start with 'Fre', and then have zero or more word characters. 
 
-`\s` matches all whitespace characters - spaces, tabs and line-breaks.
+`Fred` ✅
+`Freya` ✅
+Francis ❌
 
-`\S` matches all non-whitespace characters
+Here are a few more wildcards that can be used when you want to be more specific than using a ., \w or \W!
 
-We could use the following regex pattern to match these three strings:
+\s matches all whitespace characters - spaces, tabs and line-breaks.
 
-`/\s+abc/`
+\S matches all non-whitespace characters
+
+We could use the following regex pattern to match these three strings:
 
 ```
-' abc' ✅
-'	abc' ✅
-'    abc' ✅
-'abc' ❌
+/\s+abc/
```

Comment:
```suggestion
/\s+abc/

' abc' ✅
'	abc' ✅
'    abc' ✅
'abc' ❌
```

let's move these back inside the code block :sweat_smile: 

using code ticks (or tables, not exactly sure which) removes any spaces

![image](https://user-images.githubusercontent.com/19802557/83122794-0a9c2800-a0cc-11ea-980b-63e6cd5b6db3.png)

note: the other tables look good